content/docs/examples/multicluster/split-horizon-eds/index.md

```diff
@@ -91,34 +97,64 @@ This will be used to access the `local` pilot securely using the ingress gateway
     install/kubernetes/helm/istio > istio-auth.yaml
     {{< /text >}}
 
-1. Deploy Istio to the `local` cluster:
+1. Deploy Istio to `cluster1`:
 
     {{< text bash >}}
-    $ kubectl create --context=$CTX_LOCAL ns istio-system
-    $ kubectl create --context=$CTX_LOCAL secret generic cacerts -n istio-system --from-file=samples/certs/ca-cert.pem --from-file=samples/certs/ca-key.pem --from-file=samples/certs/root-cert.pem --from-file=samples/certs/cert-chain.pem
-    $ for i in install/kubernetes/helm/istio-init/files/crd*yaml; do kubectl apply --context=$CTX_LOCAL -f $i; done
-    $ kubectl create --context=$CTX_LOCAL -f istio-auth.yaml
+    $ kubectl create --context=$CTX_CLUSTER1 ns istio-system
+    $ kubectl create --context=$CTX_CLUSTER1 secret generic cacerts -n istio-system --from-file=samples/certs/ca-cert.pem --from-file=samples/certs/ca-key.pem --from-file=samples/certs/root-cert.pem --from-file=samples/certs/cert-chain.pem
+    $ for i in install/kubernetes/helm/istio-init/files/crd*yaml; do kubectl apply --context=$CTX_CLUSTER1 -f $i; done
+    $ kubectl create --context=$CTX_CLUSTER1 -f istio-auth.yaml
     {{< /text >}}
 
-    Wait for Istio `local` pods to come up by checking their status:
+    Wait for Istio pods on `cluster1` to come up by checking their status:
 
     {{< text bash >}}
-    $ kubectl get pods --context=$CTX_LOCAL -n istio-system
+    $ kubectl get pods --context=$CTX_CLUSTER1 -n istio-system
     {{< /text >}}
 
-### Setup the remote cluster
+1. Create the Gateway to access remote service(s):
 
-1. Export the `local` gateway address:
+    {{< text bash >}}
+    $ kubectl create --context=$CTX_CLUSTER1 -f - <<EOF
+    apiVersion: networking.istio.io/v1alpha3
+    kind: Gateway
+    metadata:
+      name: cluster-aware-gateway
+      namespace: istio-system
+    spec:
+      selector:
+        istio: ingressgateway
+      servers:
+      - port:
+          number: 15443
+          name: tls
+          protocol: TLS
+        tls:
+          mode: AUTO_PASSTHROUGH
+        hosts:
+        - "*"
+    EOF
+    {{< /text >}}
+
+    This Gateway instance is configuring an automatic routing for incoming traffic, through a dedicated port, to the relevant service
+    (SNI-based) but keeping mutual TLS all the way from the source to destination sidecars.
```

Comment:
This sentence is a little hard to understand. How about the following instead:

This `Gateway` configures a dedicated port (15443) to pass incoming traffic through to the target service specified in a request's SNI header. Mutual TLS connections will be used all the way from the source to the destination sidecar.